pdev.c

```diff
@@ -224,6 +292,8 @@ static DEVICE_ATTR_RW(fan_always_on);
 static DEVICE_ATTR_RW(fan_reduced_duty_cycle);
 static DEVICE_ATTR_RW(manual_control);
 static DEVICE_ATTR_RW(super_key_lock);
+static DEVICE_ATTR_RW(fan_boost);
```

Comment:
I am not sure a new attribute is needed, see: https://github.com/torvalds/linux/blob/f2906aa863381afb0015a9eb7fefad885d4e5a56/Documentation/ABI/testing/sysfs-class-hwmon#L289 ? Enabling fan boost is already implemented via the `pwm1_enable` attribute of the hwmon device created in `hwmon_pwm.c`, which uses `fan.c:qc71_fan_set_mode()` to turn it on, although that writes the `FAN_CTRL` register directly. I am now wondering if using the `TRIGGER_1` register might be better?